### ISSUE TITLE
如何解决抖动的问题

### ISSUE THREAD

**Issue by yslion** [1996scarlet/OpenVtuber#21]
vtuber会随着摄像头检测的点位抖动 ，比较灵敏，怎么能平滑一下下

**Comment by 1996scarlet** [comment]
Please check the following paper:
* [IECON 2023 - Improving Stability of Gaze Target Detection in Videos](https://ieeexplore.ieee.org/abstract/document/10312057)

The paper propose a Jitter loss function to capture significant jitter and impose a strong penalty during training, which empowers our model with increased stability for dynamic detection.
